### PR TITLE
Feature/trace id on event processor

### PR DESCRIPTION
**Pull request by welbert**
Added TraceId in EventConsumer's lambdas and added AwsContextId in logs as ParentId

### CODE REVIEW

**Comment by welbert** [comment]
image

File: src/Innovt.CrossCutting.Log.Serilog/ActivityEnrich.cs [hunk 1]
```diff
@@ -22,9 +22,11 @@ public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
 
         logEvent.AddOrUpdateProperty(new LogEventProperty("TraceId", new ScalarValue(activity.Id)));
```
Comment (michelmob) [inline src/Innovt.CrossCutting.Log.Serilog/ActivityEnrich.cs:23]: @welbert  Why do you want to add parent id?

**Review by michelmob — APPROVED** [review]

File: src/Innovt.CrossCutting.Log.Serilog/ActivityEnrich.cs (hunk 1 above)
Comment (welbert) [reply src/Innovt.CrossCutting.Log.Serilog/ActivityEnrich.cs:23]: In AWS case, parent id represents the request id, so add to dont lose this information

Comment (welbert) [reply src/Innovt.CrossCutting.Log.Serilog/ActivityEnrich.cs:23]: @michelmob

**Pull request merged**